builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -407,12 +418,12 @@ function install_dialog.handle_submit(this, fields)
 	end
 
 	if fields.install_all then
-		queue_download(install_dialog.package)
+		queue_download(install_dialog.package, "new")
 
 		if install_dialog.will_install_deps then
 			for _, dep in pairs(install_dialog.dependencies) do
 				if not dep.is_optional and not dep.installed and dep.package then
-					queue_download(dep.package)
+					queue_download(dep.package, "dep")
```

Comment:
`update` is not shortened either, so for consistency this should be `"dependency"`.

Reply:
i suggest to have constants instead of hardcoding string values in the code,  this is error prone

Reply:
Lua isn't statically typed, so the only way I can really do this is like so:

```lua
local REASON = setmetatable({
    dependency = "dependency",
    new = "new",
	update = "update",
}, {
	__index = function(_, key)
		error("Invalid enum value: " .. key)
	end,
})
```

is this OK?

Reply:
oh wait I'm dumb, you can just use locals